README.quickstart.vscode.md

```diff
@@ -247,6 +247,7 @@ palette_, available by selecting _View > Command Palette_ or by typing
 | `AWS: Create new SAM Template`                       | Creates a new starter SAM Template                                                                                                                                                                                                                                                                             |
 | `AWS: Deploy SAM Application`                        | Deploys a local serverless application to an AWS account. For more information, see [Deploying a Serverless Application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app) in the user guide.                                                                                      |
 | `AWS: Detect SAM CLI`                                | Checks whether the Toolkit can communicate correctly with the AWS SAM CLI that is installed.                                                                                                                                                                                                                   |
+| `AWS: Edit Credentials`                              | Opens the `~/.aws/credentials` or `~/.aws/config` file for editing.                                                                                                                                                                                                                                            |
```

Comment:
Potential idea: leverage `Commands` to generate this file ?
We could make each command name become a clickable link.

Reply:
> Potential idea: leverage Commands to generate this file ?

Definitely! Forgot that is another reason we needed the commads abstraction :)